apps/condo/domains/resident/schema/ServiceConsumer.test.js

```diff
@@ -61,48 +64,67 @@ describe('ServiceConsumer', () => {
         })
     })
 
-    describe('SoftDelete', () => {
-        it('can be soft-deleted by user with type === resident if this is his own serviceConsumer', async () => {
-            const client1 = await makeClientWithServiceConsumer()
+    describe('Read', () => {
+        it('can be read by admin', async () => {
+            const adminClient = await makeLoggedInAdminClient()
+            await makeClientWithServiceConsumer()
 
-            const [deleted] = await updateTestServiceConsumer(client1, client1.serviceConsumer.id, { deletedAt: 'true' })
-            expect(deleted.id).toEqual(client1.serviceConsumer.id)
-            expect(deleted.deletedAt).toBeDefined()
+            const objs = await ServiceConsumer.getAll(adminClient, {}, { sortBy: ['updatedAt_DESC'] })
+
+            expect(objs.length >= 1).toBeTruthy()
+            expect(objs[0].id).toBeDefined()
+            expect(objs[0].organization).toBeDefined()
+            expect(objs[0].resident).toBeDefined()
         })
 
-        it('cannot be soft-deleted by user with type === resident if this is not his own serviceConsumer', async () => {
-            const client1 = await makeClientWithServiceConsumer()
-            const client2 = await makeClientWithServiceConsumer()
+        it('can be read by user with type === resident who created it', async () => {
+            const client = await makeClientWithServiceConsumer()
+            await makeClientWithServiceConsumer() // Create second service consumer
 
-            await expectToThrowAccessDeniedErrorToObj(async () => {
-                await updateTestServiceConsumer(client1, client2.serviceConsumer.id, { deletedAt: 'true' })
+            const objs = await ServiceConsumer.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })
+
+            expect(objs.length === 1).toBeTruthy()
+            expect(objs[0].id).toMatch(client.serviceConsumer.id)
```

Comment:
Same about `[0]`

Reply:
This one should be parallelable because we use ServiceConsumer.getAll from `client` which has access-checks that restrict him to get more then 1 ServiceConsumer